src/applications/disability-benefits/2346/schemas/2346UI.js

```diff
@@ -105,6 +105,7 @@ export default {
       'ui:options': {
         classNames: 'vads-u-display--none',
         hideOnReview: true,
+        customTitle: ' ',
```

Comment:
Just confirming you meant to put a space in that empty string here 🙂

Reply:
can confirm - necessary - yolo forms system